cibuild.cmd

```diff
@@ -0,0 +1,140 @@
+@if not defined _echo echo off
+setlocal
+
+:parseArguments
+if "%1"=="" goto doneParsingArguments
+if /i "%1"=="--scope" set SCOPE=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--target" set TARGET=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--host" set HOST=%2&& shift && shift && goto parseArguments
+if /i "%1"=="--skip-bootstrap" set BOOTSTRAP_ONLY=true&& shift && goto parseArguments
+
+:: Unknown parameters
+goto :usage
+
+set CONFIGURATION=Debug-NetCore
+
+:doneParsingArguments
+
+if "%SCOPE%"=="Compile" (
```

Comment:
It'd be pretty frustrating if the script won't run because I forgot to proper case an argument value when you're doing case-insensitive comparison of the argument names above